src/modules/rover_ackermann/RoverAckermann.cpp

```diff
@@ -94,19 +103,65 @@ void RoverAckermann::Run()
 		break;
 	}
 
-	hrt_abstime now = hrt_absolute_time();
+	// Timestamps
+	hrt_abstime timestamp_prev = _timestamp;
+	_timestamp = hrt_absolute_time();
+	const float dt = (_timestamp - timestamp_prev) * 10e-6f;
+
+	if (_param_ra_max_accel.get() > 0 && _param_ra_max_speed.get() > 0
+	    && _nav_state != vehicle_status_s::NAVIGATION_STATE_ACRO) { // Acceleration slew rate
+		if (fabsf(_motor_setpoint.throttle) > fabsf(_throttle_with_accel_limit)) {
+			if (_motor_setpoint.throttle > _throttle_with_accel_limit) {
+				_throttle_with_accel_limit += (_param_ra_max_accel.get() / _param_ra_max_speed.get()) * dt;
```

Comment:
have you checked if you can use lib/slew_rate instead of implementing your own slew rating functionality?

Reply:
Maybe add some more comments, like the abs is there because it also needs to work while in reverse etc.

Reply:
implemented the library: [8222367](https://github.com/PX4/PX4-Autopilot/pull/23310/commits/8222367f84021462b2fb6b3d2b59c6839d1e53ff)